aptos-move/framework/aptos-framework/sources/configs/gas_schedule.spec.move

```diff
@@ -54,11 +54,13 @@ spec aptos_framework::gas_schedule {
         use aptos_framework::aptos_coin::AptosCoin;
         use aptos_framework::transaction_fee;
         use aptos_framework::staking_config;
+        use aptos_framework::chain_status;
 
         // TODO: set because of timeout (property proved)
         pragma verify_duration_estimate = 600;
         requires exists<stake::ValidatorFees>(@aptos_framework);
         requires exists<CoinInfo<AptosCoin>>(@aptos_framework);
+        requires chain_status::is_genesis();
```

Comment:
`set_gas_schedule` is a genesis-only function?

Moreover, this function seems to be deprecated.